root/static/scripts/annotation/AnnotationHistoryTable.js

```diff
@@ -33,22 +33,22 @@ type StateT = {
 
 type WritableStateT = {...StateT}; // this has writable properties
 
-function createInitialState(annotations) {
+function createInitialState() {
   return {
-    selectedNew: annotations[0]?.id ?? 0,
-    selectedOld: annotations[1]?.id ?? 0,
+    selectedNew: 0,
+    selectedOld: 1,
   };
 }
```

Comment:
The bug and the example mentioned in commit https://github.com/metabrainz/musicbrainz-server/pull/2564/commits/1032bd8a7a182edbf7920585b694cdd7a8042c2a make perfect sense, but I don’t understand this particular change.

Reply:
I had it, but it assumes that there are at least two annotations, whereas the previous code had fallback values in this case. See https://musicbrainz.org/release/1a80e304-aebb-4e2c-9165-ae30eec21c6d/annotations for example.

Reply:
It does, but it seemed irrelevant because there's no selection mechanism unless there's `canCompare` :) But I could always pass `canCompare` and default both to 0 if that's `false` if you feel that's better? We could not even create the state at all in that case, I guess, assuming flow won't mind...